pages/common/piper.md

```diff
@@ -0,0 +1,28 @@
+# piper
+
+> A fast, local neural text to speech system that sounds great.
+> More information: <https://github.com/rhasspy/piper>.
+
+- Output a wav [f]ile using a [m]odel (assuming config json at path to model_path + .json):
+
+`echo {{Thing to say}} | piper -m {{path/to/model.onnx}} -f {{outputfile.wav}}`
+
+- Output a wav [f]ile using a [m]odel and specifying its [c]onfig json:
+
+`echo {{Thing to say}} | piper -m {{path/to/model.onnx}} -c {{path/to/model.onnx.json}} -f {{outputfile.wav}}`
+
+- Select a particular speaker in a voice with multiple speakers:
+
+`echo {{'Warum?'}} | piper -m {{de_DE-thorsten_emotional-medium.onnx}} --speaker {{1}} -f {{angry.wav}}`
+
+- Stream output immediately to mpv (synthesizing second sentence while first sentence is spoken):
+
+`echo {{'First sentence. Second sentence'}} | piper -m {{en_GB-northern_english_male-medium.onnx}} --output-raw -f - | mpv -`
```

Comment:
```suggestion
- Stream the output to the MPV media player:

`echo {{'First sentence. Second sentence'}} | piper -m {{en_GB-northern_english_male-medium.onnx}} --output-raw -f - | mpv -`
```

Reply:
If I make this change to remove the explanation about the synthesizing, I guess I should also change the example text? For now I've gone with the one from the example two above it, which is "Thing to say"

Alternatively I could copy what the official docs does to convey the information in the example itself, which is 
```
echo 'This sentence is spoken first. This sentence is synthesized while the first sentence is spoken.' 
```
but when I did that I felt the example got too long and unwieldy.

Reply:
@spageektti made a different suggestion later on, and I went with that instead